package.json

```diff
@@ -101,6 +101,9 @@
     "git-release-notes": {
       "ejs": "^3.1.8",
       "yargs": "^17.6.2"
+    },
+    "@semantic-release/npm": {
+      "npm": "^9.4.2"
```

Comment:
version update for other packages missing

Reply:
done